PITCHME.md

```diff
@@ -424,8 +425,8 @@ Note:
 ```kotlin
 class Person(val lastName: String,
              val firstName: String,
-             val middleName: String = "") {
-    fun fullName() = "$firstName $middleName $lastName"
+             val middleName: String? = null) {
+    fun initials() = "${firstName[0]}${middleName?.get(0)?:""}${lastName[0]}"
```

Comment:
Er kanskje bedre med .first() enn [0] ?

Reply:
> Er kanskje bedre med .first() enn [0] ?

En av hver for å demonstrere at String kan betraktes som en array også?